SPECS/xfsprogs/xfsprogs.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Utilities for managing the XFS filesystem
 Name:           xfsprogs
-Version:        5.0.0
-Release:        3%{?dist}
+Version:        5.15.0
```

Comment:
Since we bumped the version here, we would also require to bump the version in the `cgmanifest.json` files as well.